todo/urls.py

```diff
@@ -0,0 +1,31 @@
+from django.urls import path
+
+from .views import (
+    TaskListView,
+    TaskCreateView,
+    TaskUpdateView,
+    TaskDeleteView,
+    TagListView,
+    TagCreateView,
+    TagUpdateView,
+    TagDeleteView,
+    TaskChangeStatusView,
+
+)
+
+urlpatterns = [
+    path("", TaskListView.as_view(), name="task-list"),
```

Comment:
Using slashes at the end of URLs: To maintain consistency, add a slash (/) to the end of routes. This avoids confusion, especially when Django automatically configures URLs.

Reply:
Some of the routes may be more explicit, such as the route to change task status.